synapse/module_api/__init__.py

```diff
@@ -146,6 +150,12 @@ class UserIpAndAgent:
     last_seen: int
 
 
+# This is sad: I'd like to express that the arguments to this Callable are
+# themselves callable. "Callback protocols" let you do this, but I couldn't find
+# a way to do so that mypy agreed was type safe.
+RegisterCallbacks = Callable[..., None]
```

Comment:
Is this used anywhere?

Reply:
... no. Can't remember where I wanted to use it. Maybe there were changes elsewhere that made this redundant.